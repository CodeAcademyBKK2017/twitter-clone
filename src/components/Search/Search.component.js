import './Search.style.css';
import React, {Component} from 'react';
import {Icon} from 'react-fa';
class Search extends Component {
  render () {
    return (
      <div className="search" >
        <input className="input" />
        <div className="search-icon">
          <Icon name='search' />
        </div>
      </div>
    );
  }    
}
export default Search;