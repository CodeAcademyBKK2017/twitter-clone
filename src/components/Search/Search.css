.search {
    flex: 0.5;
    height: 35px;
    border: 1px solid #f2eee8;
    border-radius: 14px;
    
    display: flex;
    align-items: center;
    padding: 10px 12px;
    box-sizing: border-box;
  }
  .search input[name="search"] {
    border: none;
    background: transparent;
    padding-right: 10px;
    width: 100%;
  
    box-sizing: border-box;
  }
  .search .fa {
    color: #f2eee8;
  }