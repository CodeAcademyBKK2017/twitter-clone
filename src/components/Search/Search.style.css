.search-form {
    display: inline-flex;
    padding: 5px;
    border: 1px solid lightgray;
    border-radius: 10px;
}
.search-form__input {
    border: none;
    color: darkgray;
}
.search-form__icon {
    border: none;
    color: lightgray;
}