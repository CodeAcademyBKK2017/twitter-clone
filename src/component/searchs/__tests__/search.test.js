import Search from '../search.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Search component', () => {
  it('Search: renders correctly', () => {
    const tree = renderer.create(<Search />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});