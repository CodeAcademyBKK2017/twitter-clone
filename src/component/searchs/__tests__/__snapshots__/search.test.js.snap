// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component Search: renders correctly 1`] = `
<div
  className="searchInput isFlex Center"
>
  <input
    placeholder="Search"
    type="text"
  />
</div>
`;
